src/core/cubic.c

```diff
@@ -189,6 +189,13 @@ CubicCongestionControlGetSendAllowance(
         }
 
         Cubic->LastSendAllowance = SendAllowance;
+
+        if (SendAllowance < Connection->Paths[0].Mtu) {
```

Comment:
I was also thinking of a slight modification to this:
> Only pace integral multiples of MTU, saving the rest for later. It would have the same result as this PR for sub-MTU allowances, but possible also have an effect at higher allowances.

cc @maolson-msft and @csujedihy for their thoughts

Reply:
```c
        //
        // Limit send allowance to integral multiples of MTU.
        //
        uint32_t SendCount = SendAllowance / Connection->Paths[0].Mtu;
        SendAllowance = SendCount * Connection->Paths[0].Mtu;
```

Reply:
Actually:
```c
        //
        // Limit send allowance to integral multiples of MTU (converted to the
        // datagram payload length).
        //
        uint32_t DatagramLength =
            MaxUdpPayloadSizeForFamily(
                QuicAddrGetFamily(&Connection->Paths[0].Route.RemoteAddress),
                Connection->Paths[0].Mtu);
        uint32_t SendCount = SendAllowance / DatagramLength;
        SendAllowance = SendCount * DatagramLength;
```

Reply:
This is closely related to a conversation I was having with Yi and Praveen recently- we should be restricting the congestion window to always be multiples of MTU. Currently this is not done consistently in either windows TCP or obviously msquic.